### ISSUE TITLE
torrent downloaded but deleted and not send to Radarr folder

### ISSUE THREAD

**Issue by screamjojo** [rogerfar/rdt-client#331]
**What version are you using?**
Latest docker image

**Wat OS are you running?**
Debian
**Are you using Docker or as a service?**
Docker
**Which debrid provider are you using?**
Real debrid
**Please attach a log file here with the log setting set to debug**
![docker_rdt](https://github.com/rogerfar/rdt-client/assets/71469935/86ad39b2-c19b-4d3a-a19d-478d8bfa3d4d)

Only append when I download the same file for a second time : 
1: I downloaded a file/movie with success from Radarr, file get in my movie folder and deleted in RDT, no problem
2: If I delete the file from my folder movie and refresh from radarr
3: Radarr download the same file a second time
4: File downloaded from RDT client but when finished it was deleted and always on the download folder but not send to my folder movie and missing in Radarr

**Comment by rogerfar** [comment]
I don't think this is a RDT issue as it's Radarr that is sending the delete command. 

It shows in your log:
`Not removing torrents or files for torrent` That is RDT finishing the torrent.
```
2023-10-15 11:12:34.121 +00:00 [DBG] RdtClient.Web.Controllers.QBittorrentController: Delete 3214e8a1273f953f452860c4b70adcc24bc87ec7
2023-10-15 11:12:34.121 +00:00 [DBG] RdtClient.Service.Services.QBittorrent: Delete 3214e8a1273f953f452860c4b70adcc24bc87ec7
2023-10-15 11:12:34.275 +00:00 [DBG] RdtClient.Service.Services.QBittorrent: Removing torrents from Real-Debrid and Real-Debrid Client, no files
```
This is RDT getting a command from Radarr to delete the torrent, but it won't delete the files either.
